test/helpers/test.go

```diff
@@ -80,7 +80,7 @@ var (
 	TimeoutProwActionWait = utils.GetTimeoutFromEnv("BDD_TIMEOUT_PROW_ACTION_WAIT", 5)
 
 	// EnableChatOpsTests turns on the chatops tests when specified as true
-	EnableChatOpsTests = utils.GetEnv("JX_ENABLE_TEST_CHATOPS_COMMANDS", "false")
+	EnableChatOpsTests = utils.GetEnv("BDD_ENABLE_TEST_CHATOPS_COMMANDS", "false")
```

Comment:
That is really something unrelated, right? Or is this in any form related to the issue you are describing?

Reply:
Yup, it’s a “while I was here...” change that I’m the only one who’d care about anyway. =)

Reply:
ok